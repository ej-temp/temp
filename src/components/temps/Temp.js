import React from 'react';
import PropTypes from 'prop-types';

function Temp({ name, temp }) {
  return (
    <>
      <h2>name: {name}</h2>
      <h4>Temperature: {temp}</h4>
    </>
  );
}

Temp.propTypes = {
  name: PropTypes.string.isRequired,
  temp: PropTypes.number.isRequired
};

export default Temp;
