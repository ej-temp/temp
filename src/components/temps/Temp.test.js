import React from 'react';
import { shallow } from 'enzyme';
import Temp from './Temp';

describe('Temp component', () => {
  it('renders Temp', () => {
    const wrapper = shallow(<Temp 
      name={'name'}
      temp={5}
    />);
    expect(wrapper).toMatchSnapshot();
  });
});
