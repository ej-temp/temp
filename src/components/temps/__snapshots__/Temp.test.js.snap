// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Temp component renders Temp 1`] = `
<Fragment>
  <h2>
    location: 
    name
  </h2>
  <h4>
    Temperature: 
    5
  </h4>
</Fragment>
`;
